### DISCUSSION TITLE
[Feature] Upload service on Android

### DISCUSSION THREAD

**Discussion by bencefr** [immich-app/immich#5038] [Feature Request]
### The feature

It would be great to be able to upload an image or images via an "Upload to Immich..." just like "Upload to" works with Google Photos. This would make it easy to save/share/archive images that are not stored in folders that are synchronised.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2659. Please search the existing discussions before opening a request.

**Reply by bencefr** [reply]
I did, it didn't show up.
